CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## [v0.19.6](https://github.com/chef/chef-dk/tree/v0.18.30) (2016-10-17)
+[Full Changelog](https://github.com/chef/chef-dk/compare/v0.18.30...v0.19.6)
+
+**Implemented enhancements:**
+
+- Set the always\_update\_cookbooks flag by default. [\#988](https://github.com/chef/chef-dk/pull/988) ([coderanger](https://github.com/coderanger))
+- Updated InSpec to v1.2.0
```

Comment:
Just a gem bump which we usually omit from the changelog but up to you. It just makes things look off since this one bump is not reflective of all the other bumps in the release.